packages/server/src/app.ts

```diff
@@ -205,3 +206,25 @@ export async function shutdownApp(): Promise<void> {
     rmSync(binaryStorage.replace('file:', ''), { recursive: true, force: true });
   }
 }
+
+const loggingMiddleware = (req: Request, res: Response, next: NextFunction):void=>{
+  const ctx = getRequestContext();
+  const start = new Date();
+  next();
+  const afterNext = new Date().getTime(); // Record the time after next() completes
+  const totalTime = afterNext - start.getTime(); // Calculate the time taken including the time spent in next()  
+  let userProfile: string | undefined;
+  if (ctx instanceof AuthenticatedRequestContext) {
+    userProfile = ctx.profile.reference;
+  }
+
+  ctx.logger.info('Request served', 
+  {
+    receivedAt:start,
+    requestMethodAndPath: `${req.method}: ${req.path}`,
+    Duration: `${totalTime} ms`, 
+    IP: req.ip, status: res.statusCode, 
+    UA: req.get('User-Agent'),
+    Profile: userProfile
```

Comment:
These should all have consistent casing:
```suggestion
    durationMs: `${totalTime}`, 
    ip: req.ip,
    status: res.statusCode, 
    ua: req.get('User-Agent'),
    profile: userProfile
```